packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Stats/index.tsx

```diff
@@ -40,16 +40,16 @@ const Stats: React.FC<Props> = ({ formActions, routerActions, slug, stats, total
           <Text size='16px' weight={500} color='grey600'>
             <FormattedMessage id='copy.owners' defaultMessage='Owners' />
           </Text>
-          <Text size='16px' color='white' weight={600}>
+          <Text size='16px' color='black' weight={600}>
```

Comment:
why did change the comment code? also, there should not be committed commented code